### ISSUE TITLE
FTW says 6 hours ago but last reading was 12:00 (now 16:00)

### ISSUE THREAD

**Issue by BNNorman** [ConnectedHumber/Air-Quality-Web#54]
Looking at the FTW sensor. 

Data for this sensor is delayed (phone defra) so the chart always shows X hours behind. The cron job which gathers the data runs once per hour.

The odd thing is that the defra data for 12:00 today is on the system but it is 16:00 hours now. So the "time ago" ought to show 4 hours (or 5 ; DST/BST) but not 6.

**Comment by BNNorman** [comment]
It's peddling backwards - now says 8 hours ago.
I have checked the database table and s_or_r is 14:00 today as per the popup chart.
Sounds like some form of cacheing? I'm using Firefox on Windows 10
Google Chrome shows it as 3 hours ago - which is nearer the truth.
tried other browsers like opera and chromium but they just twirl the circles.

**Comment by sbrl** [comment]
Hrm, it definitely could be caching. It might also be a rounding error. I'll have to look into this.

Other browsers should work too. I suspect we might have a bit of a performance problem to look into as well.

**Comment by sbrl** [comment]
Found the bug! It is indeed a stray cache header, I think.

**Comment by BNNorman** [comment]
Good stuff

**Comment by sbrl** [comment]
I've also fixed a CI bug too. Does it work on the beta version now?

I've also done the external rename, so feedback on that would be great too.

Internally I use Air Quality all over the place. The README and API documentation still uses the "Air Quality Web" name too. I need a new name to move everything to in that sense before I can go ahead with an internal rename.

**Comment by BNNorman** [comment]
We should talk next meetup but I think Rob may not be free to attend.

I think the names should be sensors since we are not tied to any particular
type.

**Comment by sbrl** [comment]
Hrm, it doesn't seem to have updated. It's supposed to now show "Sensor Map", not "Air Quality Map". I'll have to look into it @BNNorman.

**Comment by BNNorman** [comment]
Same here. Bloody caching

**Comment by sbrl** [comment]
Caching is great when it works....

This time though I'm suspicious of another CI bug :-/

**Comment by BNNorman** [comment]
Rather screws dynamic content.

**Comment by sbrl** [comment]
...if done incorrectly.

**Comment by sbrl** [comment]
Ah, it is showing for me now. It's odd because the title there isn't even part of the API. It's actually part of the static index.html that is served as the entry point to the web interface. Does it work for you too? Here's what I now see:

![image](https://user-images.githubusercontent.com/9929737/66699623-6d2ce900-ece0-11e9-9070-aacf75696fdb.png)

**Comment by BNNorman** [comment]
No, but im on my tablet watching a movie. Trying to clear history and cache. I do wish they'd leave stuff where i can find it.
